include/nn/crypto.h

```diff
@@ -13,10 +13,119 @@ void GenerateSha256Hash(void*, ulong, void const*, ulong);
 
 class Sha256Context;
 
-void DecryptAes128Cbc(void*, u64, void const*, u64, void const*, u64, void const*, u64);
-void EncryptAes128Cbc(void*, u64, void const*, u64, void const*, u64, void const*, u64);
-void DecryptAes128Ccm(void*, u64, void*, u64, void const*, u64, void const*, u64, void const*, u64,
-                      void const*, u64, u64);
+// Util Functions
+bool IsSameBytes(const void* data1, const void* data2, u64 size);
+Result GenerateCryptographicallyRandomBytes(void* dstBuffer, u64 dstBufferSize); // dstBufferSize max size is 0x38
```

Comment:
The comment should go above the function declaration so that it looks nicer once formatted with clang-format